docs/pages/2019_MARVEL_Psik_MaX/sections/calculations.rst

```diff
@@ -536,4 +512,17 @@ calculation.
 .. rubric:: Footnotes
 
 .. [#f1] However, to avoid duplication of KpointsData, you should first learn how to query the database, therefore we will ignore this duplication issue for now.
-.. [#f2] A process is considered active if it is either ``Created``, ``Running`` or ``Waiting``. If a process is no longer active, but terminatd, it will have a state ``Finished``, ``Killed`` or ``Excepted``.
\ No newline at end of file
+.. [#f2] A process is considered active if it is either ``Created``, ``Running`` or ``Waiting``. If a process is no longer active, but terminatd, it will have a state ``Finished``, ``Killed`` or ``Excepted``.
+
+
+Once run, AiiDA calculations are instances of the class ``CalcJob``,
```

Comment:
Is this missing a footnote tag? If not, why is it below the footnotes rubric.

Reply:
this was a leftover that I removed from above and pasted at the end of the document, sorry